src/main/java/graphql/schema/GraphQLFieldDefinition.java

```diff
@@ -184,13 +162,6 @@ public Builder deprecate(String deprecationReason) {
         }
 
         public GraphQLFieldDefinition build() {
```

Comment:
@dminkovsky has raised concern about the fetcher now being required by the field definition interface. This is my response to that. The original comments seem to have been lost in the diff update.

Hmm. I didn't quite get that. Required how?

In any case. I don't know about null. null is a valid value to return I say. I wish there was an undefined :wink:  how about throwing an exception from the build method? I've always thought of the builder as a mechanism to not just build but also validate the object being built.